src/sapi.rs

```diff
@@ -248,50 +246,55 @@ impl<'a> SpVoice<'a> {
     }
 
     pub fn set_volume(&mut self, volume: u16) {
-        unsafe { self.voice.SetVolume(min(volume, 100)) };
+        unsafe { self.voice.SetVolume(min(volume, 100)) }.unwrap();
     }
 
     #[allow(dead_code)]
     pub fn get_volume(&mut self) -> u16 {
         let mut volume = 0;
-        unsafe { self.voice.GetVolume(&mut volume) };
+        unsafe { self.voice.GetVolume(&mut volume) }.unwrap();
         volume
     }
 
-    pub fn set_alert_boundary(&mut self, boundary: winapi::um::sapi51::SPEVENTENUM) {
-        unsafe { self.voice.SetAlertBoundary(boundary) };
+    pub fn set_alert_boundary(&mut self, boundary: Speech::SPEVENTENUM) {
+        unsafe { self.voice.SetAlertBoundary(boundary) }.unwrap();
     }
 
     #[allow(dead_code)]
-    pub fn get_alert_boundary(&mut self) -> winapi::um::sapi51::SPEVENTENUM {
-        let mut boundary = 0;
-        unsafe { self.voice.GetAlertBoundary(&mut boundary) };
+    pub fn get_alert_boundary(&mut self) -> Speech::SPEVENTENUM {
+        let mut boundary = Speech::SPEVENTENUM(0);
+        unsafe { self.voice.GetAlertBoundary(&mut boundary) }.unwrap();
         boundary
     }
 
-    pub fn get_status(&mut self) -> winapi::um::sapi51::SPVOICESTATUS {
-        let mut status: winapi::um::sapi51::SPVOICESTATUS = unsafe { mem::zeroed() };
-        unsafe { self.voice.GetStatus(&mut status, null_mut()) };
+    pub fn get_status(&mut self) -> Speech::SPVOICESTATUS {
+        let mut status: Speech::SPVOICESTATUS = unsafe { mem::zeroed() };
+        unsafe { self.voice.GetStatus(&mut status, null_mut()) }.unwrap();
         status
     }
 
     fn set_notify_window_message(&mut self) {
         unsafe {
             self.voice
-                .SetNotifyWindowMessage(self.window, WM_SAPI_EVENT, 0, 0)
-        };
+                .SetNotifyWindowMessage(self.window, WM_SAPI_EVENT, WPARAM(0), LPARAM(0))
+        }
+        .unwrap();
     }
 
-    pub fn set_interest(&mut self, event: &[u32], queued: &[u32]) {
+    pub fn set_interest(&mut self, event: &[Speech::SPEVENTENUM], queued: &[Speech::SPEVENTENUM]) {
         let queued = queued
             .iter()
-            .map(|&x| winapi::um::sapi51::SPFEI(x))
+            .map(|&x| {
+                (1 << x.0) | (1 << Speech::SPEI_RESERVED1.0) | (1 << Speech::SPEI_RESERVED2.0)
```

Comment:
SPFEI macro was missing in windows crate, so I just expanded what the winapi version did.